packages/oidc/src/refresh/refreshGrant.spec.ts

```diff
@@ -114,6 +115,17 @@ describe("refreshGrant", () => {
     expect(dpopProof.payload.htu).toBe(mockIssuer().tokenEndpoint.toString());
   });
 
+  it("throws if the client identifier is undefined", async () => {
+    const client = mockClient();
```

Comment:
@NSeydoux linting issue here

Reply:
Yeah I really have to setup my IDE to catch these in the absense of precommit hooks :)